test/bitfields_packed.ll

```diff
@@ -0,0 +1,124 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-200 %t.bc -o %t.spv
+; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-200 %t.spv -o %t.spt --to-text
+; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-200 -r -emit-opaque-pointers %t.spv -o %t.bc
+; RUN: llvm-dis %t.bc -o %t.ll
```

Comment:
Please also add
; RUN: llvm-spirv --spirv-debug-info-version=ocl-100
and
; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-100

without CHECK lines - to check whether the test doesn't crash.